pkg/services/playlist/model.go

```diff
@@ -54,18 +50,18 @@ type Playlists []*Playlist
 //
 
 type UpdatePlaylistCommand struct {
-	OrgId    int64             `json:"-"`
-	UID      string            `json:"uid"`
-	Name     string            `json:"name" binding:"Required"`
-	Interval string            `json:"interval"`
-	Items    []PlaylistItemDTO `json:"items"`
+	OrgId    int64          `json:"-"`
+	UID      string         `json:"uid"`
+	Name     string         `json:"name" binding:"Required"`
+	Interval string         `json:"interval"`
+	Items    []PlaylistItem `json:"items"`
 }
 
 type CreatePlaylistCommand struct {
-	Name     string            `json:"name" binding:"Required"`
-	Interval string            `json:"interval"`
-	Items    []PlaylistItemDTO `json:"items"`
-	OrgId    int64             `json:"-"`
+	Name     string         `json:"name" binding:"Required"`
+	Interval string         `json:"interval"`
+	Items    []PlaylistItem `json:"items"`
```

Comment:
feels weird to use the "DB" type as an input, especially if we already have the DTO. Shouldn't we add the props we need to the DTO instead?

Reply:
100% agree -- the problem is that we currently accept the DB types as input, so chaning that can be seen as a breaking change so 🤷 

My focus here is getting a single clear output DTO that matches the shape used and expected by the frontend.

Reply:
not sure this is correct, we use PlaylistItemDTO as input, from what I can tell

Reply:
"from what I can tell" 🤣  yes it is very hard to tell since there are two copies, and they overlap with the internal db fields (Order, Id) so 🤷